guiEditor/src/diagram/workbench.tsx

```diff
@@ -344,11 +335,12 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
                 scene.onPointerObservable.add(panningFn, PointerEventTypes.POINTERMOVE);
                 this._panning = true;
             } else {
-                this._panning = false;
+               this._panning = false;

```

Comment:
NIT: looks like indentation is not the same as line 336 :-)

Reply:
OH! ty!